scripts/wait-for-it.sh

```diff
@@ -48,9 +54,15 @@ wait_for_wrapper()
 {
     # In order to support SIGINT during timeout: http://unix.stackexchange.com/a/57692
     if [[ $QUIET -eq 1 ]]; then
-        timeout $TIMEOUT $0 --quiet --child --host=$HOST --port=$PORT --timeout=$TIMEOUT &
+        # Modified to work with Alpine Linux
+        # Original command
+        # timeout $TIMEOUT $0 --quiet --child --host=$HOST --port=$PORT --timeout=$TIMEOUT &
+        timeout -t $TIMEOUT -s KILL $0 --quiet --child --host=$HOST --port=$PORT --timeout=$TIMEOUT &
     else
-        timeout $TIMEOUT $0 --child --host=$HOST --port=$PORT --timeout=$TIMEOUT &
+        # Modified to work with Alpine Linux
+        # Original command
+        # timeout $TIMEOUT $0 --child --host=$HOST --port=$PORT --timeout=$TIMEOUT &
+        timeout -t $TIMEOUT -s KILL $0 --child --host=$HOST --port=$PORT --timeout=$TIMEOUT &
```

Comment:
I had to modify `wait-for-it` in order to get the timeout bits working. Since we use alpine linux for builds and for CI I just modified the original and made some notes.